backend/app/modules/user/antisybil/service/initial.py

```diff
@@ -0,0 +1,89 @@
+from flask import current_app as app
+
+from typing import Optional, Tuple
+
+from datetime import datetime
+from typing import List
+
+from app.extensions import db
+from app.exceptions import ExternalApiException, UserNotFound
+from app.context.manager import Context
+from app.infrastructure import database
+from app.pydantic import Model
+from app.infrastructure.external_api.common import retry_request
+from app.infrastructure.external_api.gc_passport.score import (
+    issue_address_for_scoring,
+    fetch_score,
+    fetch_stamps,
+)
+
+
+class GitcoinPassportAntisybil(Model):
+    def get_antisybil_status(
+        self, _: Context, user_address: str
+    ) -> Optional[Tuple[float, datetime]]:
+        try:
+            score = database.user_antisybil.get_score_by_address(user_address)
+            if score is not None:
+                return score.score, score.expires_at
+            else:
+                return None
+        except UserNotFound as ex:
+            app.logger.debug(
+                f"User {user_address} antisybil status: except UserNotFound"
+            )
+            raise ex
+
+    def fetch_antisybil_status(
+        self, _: Context, user_address: str
+    ) -> (float, datetime, any):
+        score = issue_address_for_scoring(user_address)
+
+        def _retry_fetch():
+            score = fetch_score(user_address)
+            if score["status"] != "DONE":
```

Comment:
as in my previous comments, DONE can be moved to an enum / separate class like in EtherScanAPIMessages

Reply:
I don't see added value in this.

1. API we use doesn't provide a list of possible values of "status" field.
2. We just check for this value. It is not really ever propagated anywhere. Adding a type would allow us to benefit from type checking. But since we do not use it anywhere else...

Reply:
enum doesn't have to have many values, only one can be put in it; the biggest added value would be that you wouldn't have to seek every usage of "DONE" string in the code when anything in API changes (e.g DONE is replaced with SUCCESS) - you change only one place in class and it's propagated automatically; 

but if I don't convice you, it may stay as it is since it's not a blocker